Shippo/BatchShipment.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace Shippo
+{
+    [JsonObject (MemberSerialization.OptIn)]
+    public class BatchShipment : ShippoId
+    {
+        [JsonProperty (PropertyName = "object_status")]
+        public string ObjectStatus;
+
+        [JsonProperty (PropertyName = "carrier_account")]
+        public string CarrierAccount;
+
+        [JsonProperty (PropertyName = "servicelevel_token")]
+        public string ServicelevelToken;
+
+        [JsonProperty (PropertyName = "shipment")]
+        public Object Shipment;
+
+        [JsonProperty (PropertyName = "transaction")]
+        public string Transaction;
+
+        [JsonProperty (PropertyName = "messages")]
+        public List<String> Messages;
```

Comment:
This may need to be `Object` since its structure is not well defined. See https://github.com/goshippo/shippo-csharp-client/pull/18#discussion_r101866934